clients/pull_request.go

```diff
@@ -37,6 +37,13 @@ type Label struct {
 
 // Review represents a PR review.
 type Review struct {
```

Comment:
IIUC, none of the changes in `clients` are being used in this PR. Can we remove these unsused changes from this PR?